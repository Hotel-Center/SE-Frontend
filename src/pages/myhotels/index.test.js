import renderer from "react-test-renderer";
import Myhotels from ".";

describe("Manager Hotels Tab", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Myhotels />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
