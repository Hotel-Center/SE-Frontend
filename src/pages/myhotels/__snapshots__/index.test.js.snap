// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Manager Hotels Tab renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="row pt-5"
  >
    <div
      className="col-lg-3"
    >
      <div
        className="d-flex flex-column flex-shrink-0 p-3 profile-sidebar border mb-4 mb-lg-0"
      >
        <div
          className="row"
        >
          <a
            class="nav-link link-dark sidebar-items d-lg-flex mb-lg-2"
            href="/profile"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onTouchStart={[Function]}
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-colorPrimary MuiSvgIcon-fontSizeMedium css-ci5apu-MuiSvgIcon-root"
              data-testid="ModeEditOutlinedIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M3 21h3.75L17.81 9.94l-3.75-3.75L3 17.25V21zm2-2.92 9.06-9.06.92.92L5.92 19H5v-.92zM18.37 3.29a.9959.9959 0 0 0-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83c.39-.39.39-1.02 0-1.41l-2.34-2.34z"
              />
            </svg>
            <p
              className="mb-0"
            >
              Edit Profile
            </p>
          </a>
          <hr
            className="mb-2"
          />
          <a
            class="nav-link link-dark sidebar-items d-lg-flex mb-lg-2"
            href="/myhotels"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onTouchStart={[Function]}
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-colorPrimary MuiSvgIcon-fontSizeMedium css-ci5apu-MuiSvgIcon-root"
              data-testid="BedIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M21 10.78V8c0-1.65-1.35-3-3-3h-4c-.77 0-1.47.3-2 .78-.53-.48-1.23-.78-2-.78H6C4.35 5 3 6.35 3 8v2.78c-.61.55-1 1.34-1 2.22v6h2v-2h16v2h2v-6c0-.88-.39-1.67-1-2.22zM14 7h4c.55 0 1 .45 1 1v2h-6V8c0-.55.45-1 1-1zM5 8c0-.55.45-1 1-1h4c.55 0 1 .45 1 1v2H5V8z"
              />
            </svg>
            <p
              className="mb-0"
            >
              My Hotels
            </p>
          </a>
          <hr
            className="mb-2"
          />
          <a
            class="nav-link link-dark sidebar-items d-lg-flex mb-lg-2"
            href="/profile/favorites"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onTouchStart={[Function]}
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-colorPrimary MuiSvgIcon-fontSizeMedium css-ci5apu-MuiSvgIcon-root"
              data-testid="FavoriteBorderIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M16.5 3c-1.74 0-3.41.81-4.5 2.09C10.91 3.81 9.24 3 7.5 3 4.42 3 2 5.42 2 8.5c0 3.78 3.4 6.86 8.55 11.54L12 21.35l1.45-1.32C18.6 15.36 22 12.28 22 8.5 22 5.42 19.58 3 16.5 3zm-4.4 15.55-.1.1-.1-.1C7.14 14.24 4 11.39 4 8.5 4 6.5 5.5 5 7.5 5c1.54 0 3.04.99 3.57 2.36h1.87C13.46 5.99 14.96 5 16.5 5c2 0 3.5 1.5 3.5 3.5 0 2.89-3.14 5.74-7.9 10.05z"
              />
            </svg>
            <p
              className="mb-0"
            >
              Favorites
            </p>
          </a>
          <hr
            className="mb-2"
          />
          <a
            class="nav-link link-dark sidebar-items d-lg-flex"
            href="/profile/IncreaseCredit"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onTouchStart={[Function]}
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-colorPrimary MuiSvgIcon-fontSizeMedium css-ci5apu-MuiSvgIcon-root"
              data-testid="AddCardIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M20 4H4c-1.11 0-1.99.89-1.99 2L2 18c0 1.11.89 2 2 2h10v-2H4v-6h18V6c0-1.11-.89-2-2-2zm0 4H4V6h16v2zm4 9v2h-3v3h-2v-3h-3v-2h3v-3h2v3h3z"
              />
            </svg>
            <p
              className="mb-0"
            >
              Wallet
            </p>
          </a>
          <hr
            className="mb-2"
          />
          <a
            class="nav-link link-dark sidebar-items d-lg-flex"
            href="/profile/reservedrooms"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onTouchStart={[Function]}
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-colorPrimary MuiSvgIcon-fontSizeMedium css-ci5apu-MuiSvgIcon-root"
              data-testid="BookmarkBorderIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M17 3H7c-1.1 0-1.99.9-1.99 2L5 21l7-3 7 3V5c0-1.1-.9-2-2-2zm0 15-5-2.18L7 18V5h10v13z"
              />
            </svg>
            <p
              className="mb-0"
            >
              Reservations
            </p>
          </a>
        </div>
      </div>
    </div>
    <div
      className="col-lg-9"
    >
      <div
        className="row row-cols-1 row-cols-md-3 g-4"
      >
        <div
          className="MuiBox-root css-m2nmc4"
        >
          <div
            className="container"
            style={
              {
                "textAlign": "center",
              }
            }
          >
            <a
              href="/createHotel"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              <img
                data-nimg="1"
                decoding="async"
                height={100}
                loading="lazy"
                onError={[Function]}
                onLoad={[Function]}
                src="/img/pics/add_files.svg"
                style={
                  {
                    "color": "transparent",
                    "cursor": "pointer",
                  }
                }
                width={100}
              />
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
